superset/views/core.py

```diff
@@ -936,16 +937,15 @@ def clean_fulfilled_requests(session):
         return redirect('/accessrequestsmodelview/list/')
 
     def get_form_data(self):
-        # get form data from url
+        d = {}
+        # Supporting POST
+        if request.form.get('form_data'):
```

Comment:
nit: it'd be better to not `get` twice in a row. First affect to a variable and then use the var.

Reply:
fixed.